backend/kesaseteli/applications/locale/en/LC_MESSAGES/django.po

```diff
@@ -700,7 +700,7 @@ msgstr ""
 
 msgid ""
 "Kaupunki korvaa 325 euroa työnantajalle, joka palkkaa Kesäseteliin "
-"oikeutetun nuoren töihin 1.6.–15.8.2022 välisenä aikana."
+"oikeutetun nuoren töihin 1.6.–15.8.2023 välisenä aikana."
```

Comment:
I wonder if the year could be read the year from server's clock so it doesnt have to be changed every year?